api4/params.go

```diff
@@ -165,5 +168,13 @@ func ApiParamsFromRequest(r *http.Request) *ApiParams {
 		params.PerPage = val
 	}
 
+	if val, err := strconv.Atoi(r.URL.Query().Get("logs_per_page")); err != nil || val < 0 {
```

Comment:
should this be `val <= 0`?

Reply:
Your suggestion is good too.
But I think that it had better to be same as [the check of `per_page`](https://github.com/mattermost/mattermost-server/pull/7903/files/5be683aad9f6da93efd350ff7535ec5ebc4d7128#diff-0ba325e86c0672daaff52b82945b318bR163).

Should I apply `<= 0`  to `per_page` and `logs_per_page`? or apply it to `logs_per_page` only? or as it is?

Reply:
@grundleborg Thoughts?